mkt/webapps/indexers.py

```diff
@@ -30,9 +49,10 @@ def search(cls, using=None):
 
         We override this to use our patched version which adds statsd timing.
         """
-        return Search(using=using or cls.get_es(),
-                      index=cls.get_index(),
-                      doc_type=cls.get_mapping_type_name())
+        return (Search(
+            using=using or cls.get_es(), index=cls.get_index(),
+            doc_type=cls.get_mapping_type_name())
+            .extra(_source={'exclude': cls.hidden_fields}))
```

Comment:
The statsd call is inside the `Search.execute` method. The `Search` class here is not the original one. See `mkt/search/utils.py`.

So, yes, it's the same change. I wanted to replace this by just a `with statsd.timing...()` and `super()` call, but it doesn't work because the `statsd.timing` call needs to be applied to the `execute` method of `Search` class :(